airflow/cli/api/operations.py

```diff
@@ -0,0 +1,487 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+import datetime
+from typing import TYPE_CHECKING, Any
+
+import httpx
+import structlog
+
+from airflow.cli.api.datamodels._generated import (
+    AssetAliasCollectionResponse,
+    AssetAliasResponse,
+    AssetCollectionResponse,
+    AssetResponse,
+    BackfillPostBody,
+    BackfillResponse,
+    Config,
+    ConnectionBody,
+    ConnectionBulkActionResponse,
+    ConnectionBulkBody,
+    ConnectionCollectionResponse,
+    ConnectionResponse,
+    ConnectionTestResponse,
+    DAGDetailsResponse,
+    DAGResponse,
+    DAGRunCollectionResponse,
+    DAGRunResponse,
+    JobCollectionResponse,
+    PoolBulkActionResponse,
+    PoolBulkBody,
+    PoolCollectionResponse,
+    PoolPatchBody,
+    PoolPostBody,
+    PoolResponse,
+    ProviderCollectionResponse,
+    TriggerDAGRunPostBody,
+    VariableBody,
+    VariableBulkActionResponse,
+    VariableBulkBody,
+    VariableCollectionResponse,
+    VariableResponse,
+    VersionInfo,
+)
+
+if TYPE_CHECKING:
+    from airflow.cli.api.client import Client
+    from airflow.utils.state import DagRunState
+
+log = structlog.get_logger(logger_name=__name__)
+
+
+# Generic Server Response Error
+class ServerResponseError(httpx.HTTPStatusError):
+    """Server response error (Generic)."""
+
+    def __init__(self, message: str, *, request: httpx.Request, response: httpx.Response):
+        super().__init__(message, request=request, response=response)
+
+    # def_
+
```

Comment:
```suggestion
```

?

Reply:
Thanks, this one missed my eyes! I will delete this

Reply:
Removed